docs/_specification/1.2-DRAFT/structure.md

```diff
@@ -184,35 +186,38 @@ Metadata about parts of the _RO-Crate Website_ MAY be included in an RO-Crate as
 
-Payload files may appear directly in the _RO-Crate Root_ alongside the _RO-Crate Metadata File_, and/or appear in sub-directories of the _RO-Crate Root_. Each file and directory MAY be represented as [Data Entities](data-entities) in the _RO-Crate Metadata File_.
+The intention is that RO-Crates can work well with a variety of archive file formats, e.g. tar, zip, etc., and approaches to capturing file manifests and file fixity, such as [BagIt], [OCFL] and [git] (see also appendix [Combining with other packaging schemes](appendix/implementation-notes#combining-with-other-packaging-schemes)). An RO-Crate can also be hosted on the web or mainly refer to web resources, although extra care to ensure persistence and consistency should be taken for archiving such RO-Crates.
 
-An RO-Crate may also contain [Web-based Data Entities](data-entities.html#web-based-data-entities) that are not present as part of the payload and referenced using absolute URIs. These may require additional preservation measures.
 
-{% include callout.html type="tip" content="An RO-Crate [packaged with BagIt](appendix/implementation-notes#adding-ro-crate-to-bagit) may be [referencing external files](appendix/implementation-notes#referencing-external-files) which are not present in the _RO-Crate Root_ hierarchy until the BagIt has been _completed_. This method can be used for files that are large, require authentication or otherwise inconvenient to transfer with the RO-Crate, but which should nevertheless still be considered part of the _payload_." %}
 
+## Detached RO-Crate Package
 
-## Self-describing and self-contained (_Attached RO-Crates_)
+A _Detached RO-Crate Package_ is an RO-Crate, defined in an _RO-Crate Metadata Document_ without a defined root directory, where the _RO-Crate Metadata Document_ content is accessed independently (e.g. as part of a programmatic API). 
 
-RO-Crates SHOULD be self-describing and self-contained.
 
-A minimal Attached RO-Crate is a directory containing a single _RO-Crate Metadata Document_ stored as an [_RO-Crate Metadata File_](root-data-entity) `ro-crate-metadata.json`. 
+Unlike an  _Attached  RO-Crate Package_ a _Detached RO-Crate Package_ is not processed in a file-system context and thus does not carry a data _payload_ in the same sense, but may reference data deposited separately, or purely reference [contextual entities](contextual-entities). 
 
-At the basic level, an Attached RO-Crate is a collection of files and resources represented as a Schema.org [Dataset], that together form a meaningful unit for the purposes of communication, citation, distribution, preservation, etc.  The _RO-Crate Metadata Document_ describes the RO-Crate, and MUST be stored in the _RO-Crate Root_. 
+In a _Detached RO-Crate Package_ the [root data entity](root-data-entity) SHOULD have an `@id` which is a URL that resolves to the _RO-Crate Metadata Document_.
```

Comment:
Why would the `@id` of the root data entity resolve to the metadata document? They are two separate entities with different ids.

Reply:
If you have a detached crate then in many cases that will have some kind of online home at an API or a website - this is saying the the @id should point at that so it should say something like. 

In a _Detached RO-Crate Package_ the [root data entity](root-data-entity) SHOULD have an `@id` which is a URL that resolves to an online source for the _RO-Crate Metadata Document_ which may be a on the web or available over an API.

Does that make sense to you @simleo ?

Reply:
I guess I'm heavily influenced by the pre-1.2 mindset. So in a detached crate the metadata file descriptor will always have the "magic" string `ro-crate-metadata.json` as `@id`, while the root data entity's `@id` will (possibly) point to a resource where the RO-Crate metadata can actually be retrieved, right? I think changing "SHOULD have an `@id` which is a URL" to "SHOULD have an `@id` which is an absolute URL" would help make things clearer.

Reply:
+1, better be clearer in this case, although strictly speaking "URLs" are always absolute, otherwise they are "URL references"